### DISCUSSION TITLE
Dots don't matter in Gmail addresses

### DISCUSSION THREAD

**Discussion by saiSunkari19** [Web3Auth/Web3Auth#528] [Help]
Hello Team, 

We are trying to integrate web3auth into our app. But we found dot technique is not handled by your end, so this user can create multiple accounts just by changing the dot position. I was tested for my Gmail. I am able to create multiple accounts with the same mail id. Can you please check how we can able to solve this

I am attaching Gmail ref: https://support.google.com/mail/answer/7436150?hl=en#zippy=%2Cno-one-sees-your-mail

**Comment by YZhenY** [comment]
One way to get around this is to handle it on the JWT before its passed to web3auth, so that the unique identifier (often sub) that is passed does not contain dots

You need to have control of the JWT in this case, so for example it might not work directly with google if google gives out dots. Instead use google with a proxy like Auth0 or Firebase and customize that JWT to filter the dots.

Its hard for us to implement on our side because "sub" or the unique identifier that is submitted to web3auth is not exclusively email, so we can't filter it for dots on all values.

**Reply by ravaliGangasani** [reply]
What can we do on our end? Could you please send us docs if any, also we need tech support from you guys? Could you help us with that?

**Reply by saiSunkari19** [reply]
Hi @YZhenY, Thank you for the quick response. Currenlty we are using Web3Auth with a custom blockchain namespace with email_password_less.  Can you bit elaborate on how we can do that based on our requirements.

**Reply by himanshuchawla009** [reply]
I think, just remove the dots from email before passing it to web3auth sdk, let the user use email with or without dots but you always pass email without dots to web3auth sdk.

**Reply by saiSunkari19** [reply]
Yep, we will check that thank you for the suggestion.
